.github/actions/build-rpm/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN npm install && \
     npm run build && \
     npm run package
 
-FROM centos:latest
+FROM quay.io/centos/centos:stream8
```

Comment:
update to use centos stream 8, support is there till [2024-05-31](https://www.centos.org/cl-vs-cs/)